Tasks/IISWebAppManagementOnMachineGroup/task.json

```diff
@@ -1,650 +1,684 @@
-			"label": "Identity",
-			"defaultValue": "ApplicationPoolIdentity",
-			"required": true,
-			"groupName": "ApplicationPool",
-			"helpMarkDown": "Configure the account under which an application pool's worker process runs. Select one of the predefined security accounts or configure a custom account.",
-			"options": {
-				"ApplicationPoolIdentity": "Application Pool Identity",
-				"LocalService": "Local Service",
-				"LocalSystem": "Local System",
-				"NetworkService": "Network Service",
-				"SpecificUser": "Custom Account"
-			}
-		},
-		{
-			"name": "AppPoolUsername",
-			"type": "string",
-			"label": "Username",
-			"defaultValue": "",
-			"required": true,
-			"groupName": "ApplicationPool",
-			"visibleRule": "AppPoolIdentity = SpecificUser",
-			"helpMarkDown": "Provide the username of the custom account that you want to use."
-		},
-		{
-			"name": "AppPoolPassword",
-			"type": "string",
-			"label": "Password",
-			"defaultValue": "",
-			"required": false,
-			"groupName": "ApplicationPool",
-			"visibleRule": "AppPoolIdentity = SpecificUser",
-			"helpMarkDown": "Provide the password for custom account. <br/>The best practice is to create a variable in the Build or Release definition, and mark it as 'Secret' to secure it, and then use it here, like '$(userCredentials)'. <br> Note: Special characters in password are interpreted as per <a href=\"https://go.microsoft.com/fwlink/?linkid=843470\">command-line arguments</a>"
-		},
-		{
-			"name": "StartStopRecycleAppPoolName",
-			"type": "string",
-			"label": "Application pool name",
-			"defaultValue": "",
-			"required": true,
-			"visibleRule": "ActionIISApplicationPool = StartAppPool || ActionIISApplicationPool = StopAppPool || ActionIISApplicationPool = RecycleAppPool",
-			"helpMarkDown": "Provide the name of the IIS application pool."
-		},
-		{
-			"name": "AppCmdCommands",
-			"type": "multiLine",
-			"label": "Additional appcmd.exe commands",
-			"required": false,
-			"groupName": "Advanced",
-			"defaultValue": "",
-			"helpMarkDown": "Enter additional AppCmd.exe commands. For more than one command use a line separator, like <br/> list apppools <br/> list sites<br/> recycle apppool /apppool.name:ExampleAppPoolName"
-		}
-	],
-	"execution": {
-		"PowerShell3": {
-			"target": "IISWebAppManagementOnMachineGroup.ps1"
-		}
-	},
-	"messages": {
-		"InvalidSslThumbprint": "Invalid thumbprint. Length is not 40 characters or contains invalid characters.",
-		"InvalidVirtualPath": "Virtual path should begin with a /",
-		"InvalidIISDeploymentType": "Invalid IIS Deployment Type : {0}",
-		"InvalidActionIISWebsite": "Invalid action '{0}' selected for the IIS Website.",
-		"InvalidActionIISAppPool": "Invalid action '{0}' selected for the IIS Application Pool.",
-		"SSLCertWarningInvalidCharacters": "SSL Certificate thumbprint contains non-hexadecimal characters. Trimming all non-hexadecimal characters."
-	}
+    "id": "1b2aec60-dc49-11e6-9b76-63056e018cac",
```

Comment:
Any reason the entire file is shown in diff? It is hard to find only the relevant changes for review.